packages/mui-lab/src/TreeItem/treeItemClasses.ts

```diff
@@ -1,4 +1,7 @@
-import { generateUtilityClass, generateUtilityClasses } from '@mui/base';
+import {
+  unstable_generateUtilityClass as generateUtilityClass,
+  unstable_generateUtilityClasses as generateUtilityClasses,
+} from '@mui/utils';
```

Comment:
Shouldn't the lab import this from the @mui/material package?

Reply:
Makes sense.